packages/legacy/core/App/screens/PINCreate.tsx

```diff
@@ -160,135 +162,146 @@ const PINCreate: React.FC<PINCreateProps> = ({ setAuthenticated, route }) => {
     }
   }, [updatePin, PIN, PINTwo, PINOld])
 
+  const continueCreatePIN = () => {
+    setExplained(true);
+  }
+
   return (
-    <KeyboardView>
-      <View style={style.screenContainer}>
-        <View style={style.contentContainer}>
-          <PINCreateHeader updatePin={updatePin} />
-          {updatePin && (
+    <>
+    { explained ||
+      <PINExplainer continueCreatePIN={continueCreatePIN} />
+    }
+    { explained &&
```

Comment:
I believe this should be a ternary expression:
```
return explained ? (<KeyboardView>...</KeyboardView>) : (<PINExplainer continueCreatePIN={continueCreatePIN} />)
```
That way we won't need the fragment and we won't risk odd UI bugs

Reply:
Agree